examples/multimodal_vision/qwen_2_5_vl_example.py

```diff
@@ -73,7 +73,7 @@ def data_collator(batch):
     GPTQModifier(
         targets="Linear",
         scheme="W4A16",
-        ignore=["lm_head", "re:visual.*", "re:model.visual.*"],
+        ignore=["lm_head", "re:model.visual.*"],
```

Comment:
Since we want to catch both `visual.` and `model.visual.`, we can use an OR condition in our regex like
```suggestion
        # NOTE: catch anything beginning with `model.visual.` or `visual.`
        # to account for different behavior across transformers versions
        ignore=["lm_head", "re:(model\.|)visual\..*"],
```